router.scrollBehavior.js

```diff
@@ -0,0 +1,8 @@
+export default function (to, from, savedPosition) {
+  if (
+    (to.name !== 'themes-theme') &&
+    (from.name !== 'themes-theme' || from.name !== 'index')
```

Comment:
@BasVoorhoede do you know when this navigation happens? Need to test this but don't know where this is used.

Reply:
I think this is for example from `/themes` to `/themes/too-dirty`.